pkg/snapstore/s3_snapstore.go

```diff
@@ -766,3 +786,23 @@ func GetBucketImmutabilityTime(s *S3SnapStore) (bool, *int64, error) {
 
 	return false, nil, fmt.Errorf("got nil object lock configuration")
 }
+
+// IsSnapshotMarkToBeIgnored checks whether snapshot object key with given versionID is tagged to be ignored or not.
+func IsSnapshotMarkToBeIgnored(client s3iface.S3API, bucketName, key, versionID string) bool {
```

Comment:
```suggestion
// IsSnapshotMarkedToBeIgnored checks whether snapshot object key with given versionID is tagged to be ignored or not.
func IsSnapshotMarkedToBeIgnored(client s3iface.S3API, bucketName, key, versionID string) bool {
```
Minor: Marked feels right instead of Mark